.cirrus.yml

```diff
@@ -0,0 +1,9 @@
+freebsd_instance:
+  image: freebsd-12-0-release-amd64
+
+task:
+  install_script: pkg install -y
+    autoconf bash binutils coreutils e2fsprogs-libuuid
+    git gmake libunwind ncurses pkgconf
+
+  script: CFLAGS="-Wno-unused-value -DENODATA=ENOATTR" gmake
```

Comment:
Instead of defining ENODATA you should modify the code to use something else.
CFLAGS -> EXTRA_CFLAGS
What about building and running tests?

Reply:
Indeed, that was left over from a branch with some hacks/WIP to get FreeBSD running - pushed another update without it, to make it easier to see results of building upstream master on FreeBSD. Once https://github.com/pmem/issues/issues/990 is resolved I'll run the tests locally and submit another pull request to enable them.